src/flag_gems/ops/embedding.py

```diff
@@ -0,0 +1,188 @@
+import logging
+import math
+
+import torch
+import triton
+import triton.language as tl
+
+from ..utils import libentry
+
+
+@libentry()
+@triton.jit
+def embedding_kernel(
+    Y,  # pointer to the output
```

Comment:
```suggestion
    out_ptr,
```
This is not a necessary modification suggestion, but I don’t think this is an informative code comment. Why not reflect the meaning of the comment in the variable name?